radio/src/telemetry/frsky.cpp

```diff
@@ -48,7 +48,7 @@ enum FrSkyDataState {
 #endif
 };
 
-NOINLINE void processFrskyTelemetryData(uint8_t data)
+NOINLINE void processSerialData(uint8_t data)
```

Comment:
This function name here is too generic, when we speak serial on Taranis, we don't think directly of the Telemetry!